### ISSUE TITLE
atom 1.13 breaks multi-theme-applicator

### ISSUE THREAD

**Issue by vt5491** [vt5491/multi-theme-applicator#1]
Atom version 1.13 which came out on 1/1/2017 has broken multi-theme-applicator (MTA).  This is because MTA utilizes the shadow dom, which they got rid of in version 1.13.

I am investigating ways to re-factor MTA to accommodate this change.  The shadow DOM was really amenable to file level customization, so this is proving to be a challenge.  I can at least get window level theming, but my attempts at file level theming seem to always be applied at the window level (e.g. to all open files in the window).

**Comment by vt5491** [comment]
I believe I have found a way to have file level theming again.  Even better, this mechanism should also support window and pane level theming as well.  However, I need to do more testing to fully confirm this method works.

**Comment by vt5491** [comment]
v 1.0.0 which has atom 1.13 support has been released.  A scope level (new feature) has been added.  I need to do more testing and foresee making some minor changes, but I decided to push what I have now as the prior version wasn't working **_at all_**.

Mixing scopes can cause some cross-css interference, but this is a fundamental issue in a non-dom-shadowed environment.  In short CSS, which stands for *cascading* style sheets, is designed to work this way.  The only way to prevent is go to something like a shadow dom, which unfortunately was removed in atom 1.13.

The documentation still needs to be updated.
